src/components/Authentication/Login.js

```diff
@@ -25,6 +25,11 @@ const styles = {
     marginTop: "5px",
     marginRight: "8px",
     width: "19px"
+  },
+  facebookIconBtn: {
+    backgroundColor: "#3fb5a3",
```

Comment:
Can we make the facebook backgrouind colour blue and the google background colour white?

Reply:
updated